03-seeking-help.Rmd

```diff
@@ -124,6 +124,14 @@ your issue.
 > What is the difference between the `sep` and `collapse` arguments?
 > 
 
+> ## Challenge 3 {.challenge}
```

Comment:
I don't think this lesson needs more challenges. Challenge 4 is ok if the learner thinks to type ```??csv```, I'm not sure if I would think of this, perhaps more guidance (e.g. the ```read.csv()``` function is used to read data from text files...) would be helpful.